device/mellanox/x86_64-mlnx_msn2700-r0/platform.json

```diff
@@ -0,0 +1,785 @@
+                "model": "not provided",
+                "serial": "not-provided",
+                "status": "true",
+                "speed": 30,
+                "direction": "intake",
+                "target_speed": 35,
+                "tolerance": 20,
+                "led_status": "green"
+            }
+        ],
+        "fan_drawer_list": 
+        [            
+            {
+                "name": "Fan drawer 1",
+                "presence": "true",
+                "model": "not provided",
+                "serial": "not-provided",
+                "status": "true",
+                "num_fans": 2,
+                "led_status": "green"
+            },            
+            {
+                "name": "Fan drawer 2",
+                "presence": "true",
+                "model": "not provided",
+                "serial": "not-provided",
+                "status": "true",
+                "num_fans": 2,
+                "led_status": "green"
+            },            
+            {
+                "name": "Fan drawer 3",
+                "presence": "true",
+                "model": "not provided",
+                "serial": "not-provided",
+                "status": "true",
+                "num_fans": 2,
+                "led_status": "green"
+            },            
+            {
+                "name": "Fan drawer 4",
+                "presence": "true",
+                "model": "not provided",
+                "serial": "not-provided",
+                "status": "true",
+                "num_fans": 2,
+                "led_status": "green"
+            }
+        ],
+        "psu_list": 
+        [            
+            {
+                "name": "psu 1",
+                "presence": "true",
+                "model": "not provided",
+                "serial": "not-provided",
+                "status": "true",
+                "num_fans": 2,
+                "volatge": 2.1,
+                "power": 302.6,
+                "current": 150.6,
+                "powergood_status": "true",
+                "temperature": 30.125,
+                "temperature_high_threshold": 100,
+                "voltage_high_threshold": 12.1,
+                "voltage_low_threshold": 2,
+                "led_status": "green"
+            },            
+            {
+                "name": "psu 2",
+                "presence": "true",
+                "model": "not provided",
+                "serial": "not-provided",
+                "status": "true",
+                "num_fans": 2,
+                "volatge": 2.1,
+                "power": 302.6,
+                "current": 150.6,
+                "powergood_status": "true",
+                "temperature": 30.125,
+                "temperature_high_threshold": 100,
+                "voltage_high_threshold": 12.1,
+                "voltage_low_threshold": 2,
+                "led_status": "green"
+            }
+        ],
+        "thermal_mgr": [],
```

Comment:
Not sure if there's any data we can add for `thermal_mgr`. We'll need input from the Mellanox team.

Reply:
added a comment for posterity in the JSON